pkg/koordlet/runtimehooks/nri/server.go

```diff
@@ -248,13 +261,17 @@ func (p *NriServer) RemovePodSandbox(pod *api.PodSandbox) error {
 func (p *NriServer) onClose() {
 	//TODO: consider the pod status during restart
 	retryFunc := func() (bool, error) {
-		stub, err := stub.New(p, append(opts, stub.WithOnClose(p.onClose))...)
+		if p.stopped.Load() { // if set to stopped, no longer reconnect
```

Comment:
So, currently, nri server never stop, right ?

Reply:
Yes. Currently, this method is used by the testing.